package.json

```diff
@@ -69,12 +69,13 @@
     "rollup-plugin-serve": "^1.0.1",
     "rollup-plugin-typescript2": "^0.26.0",
     "ts-jest": "^25.5.1",
+    "ts-mxgraph": "git+https://github.com/process-analytics/ts-mxgraph.git#v1.0.1",
     "typescript": "^3.8.2"
   },
   "dependencies": {
     "entities": "^2.0.0",
     "fast-xml-parser": "^3.16.0",
     "json2typescript": "^1.2.5",
-    "ts-mxgraph": "git+https://github.com/process-analytics/ts-mxgraph.git#v1.0.1"
+    "mxgraph": "4.1.1"
```

Comment:
:warning: I suggest we use the `4.1.0` version to not introduce functionnal changes. ts-mxgraph@1.0.1 uses mxgraph `4.1.0`, see https://github.com/process-analytics/ts-mxgraph/blob/v1.0.1/package.json#L11
We can upgrade mxgraph later (and 4.2.0 is already available)